Source/items.cpp

```diff
@@ -3435,10 +3461,20 @@ void PrintItemPower(char plidx, ItemStruct *x)
 		sprintf(tempstr, "%i %s charges", x->_iMaxCharges, spelldata[x->_iSpell].sNameText);
 		break;
 	case IPL_FIREDAM:
+#ifdef HELLFIRE
+		if (x->_iFMinDam == x->_iFMaxDam)
+			sprintf(tempstr, "Fire hit damage: %i", x->_iFMinDam);
+		else
+#endif
 		sprintf(tempstr, "Fire hit damage: %i-%i", x->_iFMinDam, x->_iFMaxDam);
 		break;
 	case IPL_LIGHTDAM:
-		sprintf(tempstr, "Lightning hit damage: %i-%i", x->_iLMinDam, x->_iLMaxDam);
+#ifdef HELLFIRE
+		if (x->_iLMinDam == x->_iLMaxDam)
+			sprintf(tempstr, "Lightning hit damage: %i", x->_iLMinDam);
+		else
+#endif
+			sprintf(tempstr, "Lightning hit damage: %i-%i", x->_iLMinDam, x->_iLMaxDam);

```

Comment:
you added an extra tab :D

Reply:
Yeah, I felt it made more sense since hellfire also adds an else. It's always a little hard to figure out which way to go with stuff like this.

Reply:
yes but if you are going that way, there's one line where you DIDN'T add the tab :)